Packs/MailListener/Integrations/MailListenerV2/MailListenerV2_description.md

```diff
@@ -0,0 +1 @@
+Listens to a mailbox and enables incident triggering via e-mail.
```

Comment:
here should come only detailed instructions, not the integration description

Reply:
There is no detailed instructions, what should I put there?
The configuration's instructions? like below?

## Configure Mail Listener v2 on XSOAR

---

1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
2. Search for Mail Listener v2. 
3. Click __Add instance__ to create and configure a new integration instance.
    * __Name__: a textual name for the integration instance.
    * __Fetch incidents__: Whether to fetch incidents or not
    * __MailServerURL__: Mail Server Hostname / IP address
    * __port__: IMAP Port
    * __credentials__: Username and password
    * __folder__: Incoming mail folder
    * __permittedFromAdd__: Fetch mails from these senders addresses only (eg. admin@demo.com,test@demo.com)
    * __first_fetch__: First fetch time (\<number\> \<time unit\>, e.g., 12 hours, 7 days, 3 months, 1 year)
    * __limit__: The maximum number of incidents to fetch each time
    * __delete_processed__: Delete processed emails
    * __Include_raw_body__: Include raw body in incidents
    * __save_file__: Save the email .eml file
    * __TLS_connection__: Use TLS for connection (defaults to True)
    * __insecure__: Trust any certificate (not secure)
    * __incidentFetchInterval__: Incidents Fetch Interval
4. Click __Test__ to validate the connection and the authentication.

Reply:
if there isnt, just keep it empty